scripts/check-binary.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASE_URL="https://binaries.soliditylang.org/bin"
+REPO_ROOT="$(dirname "$0")/.."
+LIST_FILE=/tmp/list.json
+
+function fail() {
+    echo -e "ERROR: $@" >&2
+    exit 1
+}
+
+function check_release_version() {
+    current_version=$1
+
+    curl --silent --fail "$BASE_URL/list.json" -o $LIST_FILE
+    [[ ! -f $LIST_FILE ]] && fail "Download of release list failed:\n    [url]: $BASE_URL/list.json"
+
+    # Retrieve the latest released version
+    latest_version=$(cat $LIST_FILE | jq --raw-output ".latestRelease")
+    release_version=$(cat $LIST_FILE | jq --raw-output ".releases | .[\"$latest_version\"]" | sed --regexp-extended --quiet 's/^soljson-v(.*).js$/\1/p')
```

Comment:
They're both the latest version, aren't they? For me this naming would be clearer:
```suggestion
    latest_version_short=$(cat $LIST_FILE | jq --raw-output ".latestRelease")
    latest_version_long=$(cat $LIST_FILE | jq --raw-output ".releases | .[\"$latest_version_short\"]" | sed --regexp-extended --quiet 's/^soljson-v(.*).js$/\1/p')
```